ui/src/routes/_layout/create-organization.tsx

```diff
@@ -66,10 +64,13 @@ const CreateOrganizationPage = () => {
       });
     },
     onError(error: ApiError) {
-      toast({
-        title: error.message,
+      toast.error(error.message, {
         description: <ToastError error={error} />,
-        variant: 'destructive',
+        duration: Infinity,
+        cancel: {
+          label: 'Cancel',
```

Comment:
Is this what the user sees / needs to click in order to get rid of the toast? If so, I'd prefer "Dismiss" over "Cancel".